config/pmd.xml

```diff
@@ -298,14 +298,6 @@
     | //ClassOrInterfaceDeclaration[@Image='Main']"/>
     </properties>
   </rule>
-  <rule ref="category/java/design.xml/ImmutableField">
```

Comment:
We do not need this suppression. The CliOptions are already suppressed in `pmd-main.xml`